xdsl/interactive/passes.py

```diff
@@ -1,15 +1,13 @@
+from collections.abc import Callable
 from typing import NamedTuple
 
 from xdsl.context import MLContext
 from xdsl.dialects import builtin, get_all_dialects
+from xdsl.ir import Dialect
 from xdsl.passes import ModulePass, PipelinePass
-from xdsl.transforms import get_all_passes
 from xdsl.transforms.mlir_opt import MLIROptPass
 from xdsl.utils.parse_pipeline import PipelinePassSpec
 
-ALL_PASSES = tuple(sorted((p_name, p()) for (p_name, p) in get_all_passes().items()))
```

Comment:
the immutability is why this was a tuple also

Reply:
Ah, I assumed it was for the ordered Ness (python dicts are now ordered). Happy to make both tuples or make them private or both